templates/csharp/sso-tab-ssr/GettingStarted.md

```diff
@@ -14,6 +14,9 @@ New to Teams app development or Teams Toolkit? Learn more about
 Teams app manifests, deploying to the cloud, and more in the documentation 
 at https://aka.ms/teams-toolkit-vs-docs
 
+This sample is configured as interactive server-side rendering.
+For more details about Blazor render mode, please refer to https://learn.microsoft.com/aspnet/core/blazor/components/render-modes.
+
```

Comment:
update to:
"...... please refer to [ASP.NET Core Blazor render modes | Microsoft Learn](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0)"

Reply:
Updated.